service/src/main/scala/com/yahoo/maha/service/calcite/MahaCalciteSqlParser.scala

```diff
@@ -252,7 +252,10 @@ case class DefaultMahaCalciteSqlParser(mahaServiceConfig: MahaServiceConfig) ext
       case SqlKind.BETWEEN =>
         ArrayBuffer.empty += BetweenFilter(toLiteral(operands(0)), toLiteral(operands(1)), toLiteral(operands(2))).asInstanceOf[Filter]
       case SqlKind.LIKE =>
-        ArrayBuffer.empty += LikeFilter(toLiteral(operands(0)), toLiteral(operands(1))).asInstanceOf[Filter]
+        if (sqlBasicCall.getOperator.getName.contains("NOT"))
```

Comment:
is there any cleaner way to do this in sql cell ?
NOT can be lowercase as well.

Reply:
I used to try to use the `negated` field in SqlOperator, but in SqlOperator class this field does not exist. And tried to search online as well, only find negate() method which is not fit into this case
<img width="1680" alt="Screen Shot 2021-07-20 at 6 21 51 PM" src="https://user-images.githubusercontent.com/42597424/126415618-a5efa188-2c56-4925-86f0-4f26b564f6fc.png">

Reply:
just found out a clearer way to do that:
`sqlBasicCall.getOperator.asInstanceOf[SqlLikeOperator].isNegated`